config/routes.rb

```diff
@@ -62,7 +62,7 @@
         end
       end
 
-      jsonapi_resources :npq_profiles, only: [:create]
+      resources :npq_profiles, only: [:create], path: "npq-profiles"
```

Comment:
what was the old path? Will it need the change on the client?

Reply:
endpoint has not changed. `jsonapi_resources` under the hood converts to a hyphen for the path